eng/test-build-correctness.ps1

```diff
@@ -44,10 +44,10 @@ try {
 
   # Verify no PROTOTYPE marker left in main
   if ($env:SYSTEM_PULLREQUEST_TARGETBRANCH -eq "main") {
-    Write-Host "Checking no PROTOTYPE markers in compiler source"
-    $prototypes = Get-ChildItem -Path src/Compilers/*.cs, src/Compilers/*.vb,src/Compilers/*.xml -Recurse | Select-String -Pattern 'PROTOTYPE' -CaseSensitive -SimpleMatch
+    Write-Host "Checking no PROTOTYPE markers in source"
+    $prototypes = Get-ChildItem -Path src/*.cs, src/*.vb, src/*.xml -Recurse | Select-String -Pattern 'PROTOTYPE' -CaseSensitive -SimpleMatch
```

Comment:
```suggestion
    $prototypes = Get-ChildItem -Path src, eng, scripts -Exclude *.dll,*.exe,*.pdb,*.xlf -Recurse | Select-String -Pattern 'PROTOTYPE' -CaseSensitive -SimpleMatch
```
Now that we're searching "everything" I think that will be a more effective search. It will catch project files, markdown, random text docs, etc ... I did a quick test on my machine to make sure it didn't include anything unexpected. 

```
C:\Users\jaredpar\code\roslyn>  Get-ChildItem -Path src, eng, scripts -Exclude *.dll,*.exe,*.pdb,*.xlf -Recurse | group-object -property extension -noelement | sort-object Count -descending

Count Name
----- ----
10508 .cs
 3488 .vb
 3002
  201 .csproj
  146 .txt
   61 .resx
   58 .ps1
   53 .json
   47 .vbproj
   44 .xaml
   37 .yml
   36 .il
   29 .sln
   29 .props
   28 .targets
   25 .sh
   25 .netmodule
   24 .shproj
   24 .projitems
   16 .config
   15 .cmd
   15 .md
   14 .winmd
   10 .snk
    9 .rsp
    9 .xml
    8 .pkgdef
    7 .vsixmanifest
    6 .vsdconfigxml
    6 .xsd
    6 .xunit
    6 .bat
    5 .patch
    5 .editorconfig
    4 .csx
    4 .vsct
    4 .bionic
    4 .cpp
    3 .ruleset
    3 .Desktop
    3 .rtf
    3 .png
    3 .nuspec
    3 .res
    3 .xenial
    3 .globalconfig
    3 .mod
    <omitted rest /> 
```

Reply:
Thanks for the suggestion